src/style.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Sofia+Sans&display=swap');
+*{
+    font-family: 'Sofia Sans', sans-serif;
+}
+
+.reg-log-ctn{
```

Comment:
This is not in my skill zone to comment on, but I have to say it. It's probably smart move to start using tailwind or bootstrap. This takes your time and effort to write, and there are more important things for you to learn and experience than CSS.